rstfinder/rst_parse.py

```diff
@@ -172,8 +172,8 @@ def from_constituency_trees(tree_strings, segmenter, rst_parser):
     """
     # return empty output if the list of trees is empty
     if len(tree_strings) == 0:
-        logging.warning(f"The input contained no trees.")
-        return [], []
+        logging.warning("The input contained no trees.")
```

Comment:
Thanks for addressing the issue. Here is another place need to be fixed. 
I should mention it in the issue. Sorry for that. 

https://github.com/EducationalTestingService/rstfinder/blob/develop/rstfinder/rst_parse.py#L192

Reply:
Once the exception is raised, the return statement is never executed. That’s why I removed this because this code is unreachable.